docs/SEISMIC_SITE_NAMING_CONVENTIONS.md

```diff
@@ -0,0 +1,29 @@
+# Station and Location Code Changes
+
+The conventions GeoNet use for site and location code changes are outlined in this document.
+
+The ISC guidelines state "because of the need for accurate station positions for hypocenter location programs, a new international code is assigned if a station is moved more than one (1) kilometer from the previous location. If the move is less than one (1) kilometer, a new code will be assigned if requested by the operator."
+
+We note that these guidelines are geared towards global seismic networks, and that our network is regional and local in its nature. Where in the global case a sensor change within 1 km does not change derived earthquake solutions much, in the case of networks designed for regional and local-scale earthquake studies, such a change in position without appropriate metadata changes would cause gross error in derived earthquake solutions.
+
+## Stations, Sites, Location Codes
+
+The designation practices of station, site, and location codes are described in the sister document `SEISMIC_CHANNEL_NAMING_CONVENTIONS.md`.  In short, station codes are designed to convey an idea of where the station is located, site codes denote different sensor installation locations at a station, and location codes describe the type(s) of sensor installed at a site.
+
+A station can have many sites, and a site can have many location codes. Ideally, each site will only have one location code associated with it, but this is often not the case. There may be multiple location codes at a site when:
+1. There are sensors of multiple types installed at a site, e.g. a weak motion sensor and a strong motion sensor.
+1. There are multiple sensors of the same type installed at a site, e.g. two weak motion sensors.
+1. The subtype of a sensor has changed, e.g. a short period sensor has been swapped for a broadband sensor, but both are weak motion sensors.
```

Comment:
There's really no difference between a short-period and a broadband (both weak motion velocity), so no need to change location code.